src/extensions/endpoints/unadopted-probes/test/index.test.ts

```
@@ -0,0 +1,234 @@
+describe('local-adoption endpoint', () => {
+	const sandbox = sinon.createSandbox();
+
+	const dbStub = {
+		where: sandbox.stub(),
+		whereNull: sandbox.stub(),
+		whereNotNull: sandbox.stub(),
+		select: sandbox.stub(),
+		orWhereRaw: sandbox.stub(),
+		whereRaw: sandbox.stub(),
+		first: sandbox.stub(),
+		then: sandbox.stub(),
```

Biome (2.3.13)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)